### ISSUE TITLE
Tickler: Clear Filter and Reload Buttons

### ISSUE THREAD

**Issue by yingbull** [carlos-emr/carlos#264]
## Summary

Add **Clear Filter** and **Reload** buttons to the Tickler List filter bar that allow users to quickly reset all filters to defaults and refresh the data.

**Blocked by:** #247 (Bootstrap 5, Font Awesome 6.x, Theme)
**Pattern:** Same as consultation list improvements

## Features

1. **Clear Filter Button**
   - Resets all filters to defaults:
     - Status = "A" (Active)
     - MRP = "all"
     - Creator = "all"
     - Assigned To = "all"
     - Priority = "all"
     - Date range = current year
   - Reloads list with cleared filters

2. **Reload Button**
   - Refreshes tickler list with current filter values
   - Useful after data changes in another window

## UI Design

- Clear Filter: `btn-outline-secondary btn-sm` with filter-reset icon
- Reload: `btn-outline-primary btn-sm` with refresh icon
- Responsive: icons only on mobile screens

### HTML Buttons
```jsp
<div class="tickler-filter-actions d-inline-flex align-items-center ms-3">
    <button type="button" id="clearFiltersBtn" class="btn btn-outline-secondary btn-sm me-2"
            onclick="clearAllFilters()" title="Reset all filters">
        <i class="fa-solid fa-filter-circle-xmark me-1"></i>
        <span class="d-none d-md-inline">Clear Filters</span>
    </button>
    <button type="button" id="reloadBtn" class="btn btn-outline-primary btn-sm"
            onclick="reloadTicklers()" title="Refresh list">
        <i class="fa-solid fa-rotate me-1"></i>
        <span class="d-none d-md-inline">Reload</span>
    </button>
</div>
```

### Vanilla JS Functions
```javascript
function clearAllFilters() {
    document.getElementById('ticklerview').value = 'A';
    document.getElementById('mrpview').value = 'all';
    document.getElementById('providerview').value = 'all';
    document.getElementById('assignedTo').value = 'all';
    // ... reset other filters
    reloadTicklers();
}

function reloadTicklers() {
    if ($.fn.DataTable.isDataTable('#ticklerTable')) {
        $('#ticklerTable').DataTable().ajax.reload();
    } else {
        document.querySelector('form[name="ticklerform"]').submit();
    }
}
```

## Implementation Checklist

- [ ] Add Clear Filter button HTML
- [ ] Add Reload button HTML
- [ ] Add `clearAllFilters()` JavaScript function
- [ ] Add `reloadTicklers()` JavaScript function
- [ ] Add CSS for button layout and responsive behavior
- [ ] Add i18n messages for button labels and titles

---
*Generated with Claude Code*
